core/templates/pages/contributor-dashboard-page/modal-templates/translation-modal.component.ts

```diff
@@ -113,7 +142,10 @@ export class TranslationModalComponent {
     this.HTML_SCHEMA = {
       type: 'html',
       ui_config: {
-        hide_complex_extensions: true,
+        // If this is made true, then the translation can not be validated
+        // properly since there can be complex extensions in the translatable
+        // text as complex extensions are not hidden in the exploration editor.
```

Comment:
```suggestion
        // If this is made true, then the translation cannot be validated
        // properly since contributors will not be able to view and translate
        // complex extensions.
```

Reply:
Done